specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2019-08-01/managedClusters.json

```diff
@@ -1672,6 +1672,7 @@
           "description": "Desired outbound IP resources for the cluster load balancer."
         },
         "effectiveOutboundIPs": {
+          "readOnly": true,
```

Comment:
You also should change/fix this in the branch Fuming has outstanding.

Reply:
(Will need to be a separate PR)

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/33046

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/33045

Reply:
The change has been merged.